lib/symboldatabase.cpp

```diff
@@ -5310,6 +5310,13 @@ void SymbolDatabase::setValueTypeInTokenList()
                 }
             }
 
+            // function style cast
+            else if (Token::Match(tok->previous(), "bool|char|short|int|long|float|double")) {
```

Comment:
what do you think about using `Token::isStandardType()`?

Reply:
I tried that but got a test crash on garbage code with a void in it.

Reply:
I fixed the crash with isStandardType().